### DISCUSSION TITLE
Making detailed bookmark icons visible at a greater map zoom level?

### DISCUSSION THREAD

**Discussion by palzkillm** [organicmaps/organicmaps#5177] [Q&A] [answered]
Hello!
I'm creating bookmarks with individual icons through KML, see code example below. Those icons are visible nicely. But if you zoom out of the map, at a certain zoom level, they are automatically replaced by the default star icon.

Question: Is there a way to make the individual icons visible at a greater zoom level?
I tried `<mwm:scale>12</mwm:scale>`, but that didnt work.

Code example for a KML Placemark
`<Placemark>
	<name><![CDATA[Name]]></name>
	<description><![CDATA[Desctiption]]></description>
	<styleUrl>#placemark-brown</styleUrl>
	<Point><coordinates>10.233483, 53.690067,0</coordinates></Point>
	<ExtendedData xmlns:mwm='https://omaps.app'>
		<mwm:icon>Shop</mwm:icon>
	</ExtendedData>
</Placemark>
`
Thanks,
Marco

**Comment by biodranik** [comment, marked as answer]
It is likely hard-coded in the code. Do you think that the map will look better with them?

**Reply by palzkillm** [reply]
I dont know wether it would look better - looks are subjective. But it would assist me better in finding my destinations. After all, I'm not using Organic Maps because I think the map looks good - I'm using it because its fast as a lightning, works offline, and is relatively flexible. :)

Anyways, closing this, it's not a pressing issue for me.
